src/com/facebook/buck/android/MergeAndroidResourcesStep.java

```diff
@@ -216,29 +236,55 @@ public StepExecutionResult execute(ExecutionContext context)
               filesystem,
               useOldStyleableFormat);
 
-      // If a resource_union_package was specified, copy all resource into that package,
-      // unless they are already present.
-      if (unionPackage.isPresent()) {
+      ImmutableSet.Builder<String> requiredPackages = ImmutableSet.<String>builder();
+
+      if (skipPrebuiltRDotJava) {
```

Comment:
Similar feedback here -- there's a lot of code duplication here that should be eliminated.

Reply:
remove code duplication